tomviz/PipelineView.cxx

```diff
@@ -330,6 +332,11 @@ void PipelineView::contextMenuEvent(QContextMenuEvent* e)
     hideAction = contextMenu.addAction("Hide");
     showAction = contextMenu.addAction("Show");
 
+    if (selectedIndexes().size() == 1) {
```

Comment:
The action is never added to the context menu, but removing the if condition makes it appear